msdtc/map.jinja

```diff
@@ -9,6 +9,7 @@
   key:values here that differ from whats in defaults.yml
 ##}
 {% set osrelease_map = salt['grains.filter_by']({
+    '2012ServerR2': {},
```

Comment:
You'd be better off in the long run using the default param or just getting rid of the osrelease map all together.